src/components/product/PriceOfferModal/index.tsx

```diff
@@ -0,0 +1,154 @@
+import { Input, Divider, Radio, IconButton } from '@offer-ui/react'
+import type { ReactElement } from 'react'
+import { useState } from 'react'
+import { Styled } from './styled'
+import type { OfferForm, PriceOfferModalProps } from './types'
+import { TRADE_METHOD } from '@constants'
+
+const MOCK_CITY = [
+  { code: 1, name: 'Seoul' },
+  {
+    code: 2,
+    name: 'Deajeon'
+  },
+  {
+    code: 3,
+    name: 'Incheon'
+  }
+]
+
+const initialOfferForm = {
+  tradeMethod: 0,
+  tradeLocation: {
+    city: 0,
+    country: 0,
+    town: 0
+  }
+}
+
+export const PriceOfferModal = ({
+  handleClickOffer,
+  ...props
+}: PriceOfferModalProps): ReactElement => {
+  const [offerForm, setOfferForm] = useState<OfferForm>(initialOfferForm)
+  const isTradeDirect = offerForm.tradeMethod !== 8
+  const canOffer =
+    !!offerForm.price &&
+    !!offerForm.tradeMethod &&
+    !!offerForm.tradeLocation.city &&
+    !!offerForm.tradeLocation.country &&
+    !!offerForm.tradeLocation.town
+
+  const handleChangeForm = (
+    name: 'price' | 'tradeMethod',
+    value: string
+  ): void => {
+    const numberValue = Number(value.split(',').join(''))
+
+    setOfferForm(prev => ({
+      ...prev,
+      [name]: numberValue
+    }))
+  }
+
+  const handleChangeSelectBox = (
+    name: keyof OfferForm['tradeLocation'],
+    value: number
+  ): void => {
+    setOfferForm(prev => ({
+      ...prev,
+      tradeLocation: {
+        ...prev.tradeLocation,
+        [name]: value
+      }
+    }))
+  }
+
+  return (
+    <Styled.PriceOfferModal {...props}>
+      <Styled.Header>
+        <Styled.CloseIconWrapper>
+          <IconButton color="grayScale30" icon="close" size={24} />
+        </Styled.CloseIconWrapper>
+        <Styled.Title>가격을 제안해볼까요?</Styled.Title>
+        <Styled.Description>
+          가격이 마음에 든다면, 연락이 올거에요!
+        </Styled.Description>
+      </Styled.Header>
+      <Styled.Body>
+        <Styled.FormItemContainer>
+          <div>제안 가격</div>
+          <Input
+            isPrice
+            placeholder="제안할 가격을 적어주세요"
+            value={offerForm.price}
+            onChange={(e): void => handleChangeForm('price', e.target.value)}
```

Comment:
커링으로 하나의 핸들러로 처리하는 형태도 좋을 것 같습니다.
위, 아래 핸들러 함수 확인하지 않고 위에서 로직 파악만 할 수 있어서 좋은 것 같습니다!

- TO-BE

```ts
const handleChangeForm: HandleChangeForm = name => e => {
    const formattedValue = Number(e.target.value.split(',').join(''))

    setOfferForm(prev => ({
      ...prev,
      [name]: formattedValue
    }))
  }
```

```suggestion
            onChange={handleChangeForm('price')}
```

Reply:
요 함수는 input, form에 대한 changeEvent를 동시에 핸들링하면서 만들어졌는데요 !
input, form 동작에 따라 다르게 실행되야 하는 부분이 있어 두가지 함수를 나누었습니다 ! 
아래 커밋을 확인하시면 됩니다 !

[반영](https://github.com/price-offer/offer-fe/pull/106/commits/983004a8661137d4ecb1cb138f53c818964777cb) 완료!